src/utils/_colors.scss

```diff
@@ -12,10 +12,16 @@
 // $c-primary-dark: color-variation($c-primary, -2) -> 2 steps darker
 // $c-primary-darkest: color-variation($c-primary, -4) -> 4 steps darker
 
+$color-variation-dark: black;
```

Comment:
I think you need a  ! default here to be able to customise in your theme

Reply:
ups, right, done!